__tests__/mutableAtom.test.tsx

```diff
@@ -0,0 +1,398 @@
+import React from 'react'
+import { act, render, renderHook, waitFor } from '@testing-library/react'
+import { atom, getDefaultStore, useAtom, useAtomValue, useSetAtom } from 'jotai'
+import assert from 'minimalistic-assert'
+import { mutableAtom } from '../src/mutableAtom'
+import type { ProxyState } from '../src/mutableAtom'
+
+it('should be defined on initial render', async () => {
+  expect.assertions(1)
+  const mutableCountAtom = mutableAtom(0)
+
+  let countProxyIsDefined = false
+  let runCount = 0
+  function Test() {
+    const countProxy = useAtomValue(mutableCountAtom)
+    if (runCount === 0) {
+      countProxyIsDefined = !!countProxy
+    }
+    runCount++
+    return null
+  }
+
+  render(<Test />)
+  await waitFor(() => assert(runCount > 0))
```

Comment:
By the way, I would simply use expect, is `assert` is simply for a shorthand?
```suggestion
  await waitFor(() => expect(runCount > 0).toBe(true))
```